src/ServiceControl.Audit/Persistence/PersistenceLifecyleManager.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace ServiceControl.Audit.Persistence
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Hosting;
+
+    class PersistenceLifecyleManager : IHostedService
```

Comment:
I am not a big fan of that name. We have PersistenceLifecycle and PersistentLifecycleManager. How about we call this PersistentLifecycleHostedService? (most hosted services have that in their names in SC codebase)

Reply:
Agreed, renamed to `PersistenceLifecycleHostedService`